group_vars/all/vlans.yaml

```diff
@@ -1,17 +1,323 @@
----
 vlans:
-  100:
-    name: "VLAN 100"
-    description: "This is the description of example vlan 100"
+  10:
+    name: "CSAIL-MAIN"
+    description: "Openshift public facing services (horizon, keystone)"
```

Comment:
sorry, just write "Various Openshift clusters" for this. There's no horizon or keystone for openshift, that's only for openstack.

Reply:
Sure, just updated